lib/AmbientOcclusion.js

```diff
@@ -99,21 +99,25 @@ function bakeAmbientOcclusion(aoOptions) {
         if (!options.triangleCenterOnly) {
             NodeHelpers.forEachPrimitiveInScene(gltf, scene, raytraceOverTriangleSamples, parameters);
         }
-        bakeToVertices({
-            gltf : gltf,
+        bakeToVertices(gltf, {
             scene : scene,
             aoBufferByPrimitive : raytracerScene.aoBufferByPrimitive,
             shaderMode : options.shaderMode,
             ambientShadowContribution : options.ambientShadowContribution
         });
     }
-}
+};
+
+AmbientOcclusion.generateOptions = generateOptions;
```

Comment:
Are these functions used outside of testing?  If not, then get rid of the `AmbientOcclusion` object and just store these directly on `bakeAmbientOcclusion` (and make `bakeAmbientOcclusion` the only thing you export).